tests/api_tests/users/test_views.py

```diff
@@ -229,110 +232,118 @@ def tearDown(self):
         super(TestUserRoutesNodeRoutes, self).tearDown()
         Node.remove()
 
-    def test_path_Users_User_id_Nodes_user_not_logged_in(self):  #~WORK
-        url = "/{}users/{}/nodes/".format(API_BASE, self.user_one._id)
-        res = self.app.get(url)
-        node_json = res.json['data']
+    def test_get_200_path_users_me_userone_logged_in(self):
+        url = "/{}users/me/".format(API_BASE)
+        res = self.app.get(url, auth=self.auth_one)
+        assert_equal(res.status_code, 200)
 
-        ids = [each['id'] for each in node_json]
-        assert_in(self.public_project_user_one._id, ids)
```

Comment:
Why were all these asserts removed? These were useful checks. Just getting a status code is not enough.